distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1532,3 +1545,9 @@ run_vectorized_binary_case_text_op left op other case_sensitivity fallback =
                passing the locale to it.
                See: https://www.pivotaltracker.com/n/projects/2539304/stories/184093260
             run_vectorized_binary_op left Nothing fallback other
+
+## PRIVATE
+   This will be replaced once we change parse to rely on `Value_Type` instead.
+ensure_valid_value_type type ~action =
+    expected_types = [Auto, Integer, Decimal, Date, Date_Time, Time_Of_Day, Boolean]
+    if expected_types.contains type . not then Error.throw (Illegal_Argument.Error "Unsupported target type "+type.to_text+".") else action
```

Comment:
Shall we move this to the `Value_Type` module and rename to `ensure_valid_parse_target`?